cli/src/features/test-conf/config.yml

```diff
@@ -3,42 +3,57 @@ image: 'jembi/go-cli-test-image'
 logPath: /tmp/logs
 
 packages:
-  - id: core
-    name: Core package
-  - id: client
-    name: OpenCR Client Registry
-  - id: dashboard-visualiser-jsreport
-    name: Dashboard Visualiser - JS Report
+  - core
+  - client
+  - dashboard-visualiser-jsreport
 
 customPackages:
   - id: disi-on-platform
     path: "git@github.com:jembi/disi-on-platform.git"
     sshKey: ./id_rsa
     sshPassword: ./id_rsa_password.txt
+  - id: cares-on-platform
+    path: ./features/cares-on-platform
```

Comment:
The packages that you're pointing to should exist in source-control. My tests failed locally because features/cares-on-platform doesn't exist on this branch. Please also make sure to give it a better name, like test-custom-packages or something.